### ISSUE TITLE
Extends bpkg compatible packages 

### ISSUE THREAD

**Issue by mbl-35** [bpkg/bpkg#100]
It would be great if the install et getdeps process could be extended to find a bpkg repo.
I mean, not only to search about a **package.json** or **Makefile**, but also a **bpkg.json** metadata file decription file (set before searching for package.json).
This is actually the only restriction (with @hydra1983 new features for gitlab target) why I can't use bpkg in a project (with composer and npm...).
I cant use npm and bpkg on a same project...
What was the reasons to have packages.json name (used by npm) ?

**Comment by Potherca** [comment]
> What was the reasons to have packages.json name (used by npm) ?

There's a discussion answering this question here: https://github.com/bpkg/bpkg/issues/17

I'd also like to have `bpkg.json` rather than `packages.json`.

**Comment by mbl-35** [comment]
Thank's for that informations. 
Is that project still active ? There is only few commits still one year...

**Comment by jwerle** [comment]
@mbl-35 yes it sure is. I'm happy to send invites for those interested in actively driving this project forward. I'm pretty occupied these days, but still very happy to participate in the conversation!
